.gitignore

```diff
@@ -4,7 +4,8 @@ backstop/results/*
 !backstop/results/bitmaps_reference
 
 # Built Files
-lib
+scss
```

Comment:
Review this file, maybe.. I think paths needs to be udpated, cause after running build, all minified files are shown as new untracked files

Reply:
Updated

Reply:
I think that one not enough.. have you checked what happens with test images? They all showed as untracted files after tests run..

Reply:
Should I move these things to a second gitignore inside `packages/itwinui-css`? Would be more reliable there

Reply:
Oh, i see we have multiple gitignores in monorepo in react, so its fine..
I guess just needed to add `*` at the start, so it would pick those anywhere.